tools/mod/go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/hexfusion/schwag v0.0.0-20211117114134-3ceb0191ccbf
 	github.com/mdempsky/unconvert v0.0.0-20200228143138-95ecdbfc0b5f
 	github.com/mgechev/revive v1.2.1
-	github.com/mikefarah/yq/v4 v4.24.2
+	github.com/mikefarah/yq/v4 v4.24.5
```

Comment:
Can we bump the version in a separate & dedicated PR?

Reply:
Done